src/Illuminate/Routing/Router.php

```diff
@@ -387,6 +383,8 @@ public function group(array $attributes, $routes)
 
             array_pop($this->groupStack);
         }
+
+        return $this;
```

Comment:
You might have to rebase, this change was already made here:
https://github.com/laravel/framework/pull/44825/files

Reply:
Commit changed